src/tsHelper.js

```diff
@@ -9,6 +9,10 @@ module.exports.getImportsForFile = function getImportsForFile(file, srcRoot) {
         .map(importedFile => importedFile.fileName)
         .filter(fileName => !/^vs\/css!/.test(fileName)) // remove css imports
         .filter(x => /\//.test(x)) // remove node modules (the import must contain '/')
+        .filter(x => /\@/.test(x) == false) // remove node modules with @
+        .filter(x => /next/.test(x) == false) // filter next/xyz imports
+        .filter(x => /lodash/.test(x) == false) // filter lodash/xyz imports
+        .filter(x => !/.svg|.js/.test(x)) // filter .svg and .js imports
```

Comment:
Most of these filters are intended to ignore packages in `node_modules`. Some nested imports like `import thing from lodash/thing` weren't excluded by the catch-all filter provided by the script: `.filter(x => /\//.test(x))`. Try removing these filters and take a look at the errors. There may be better ways to handle this